data/campaigns/Heir_To_The_Throne/scenarios/20b_Underground_Channels.cfg

```diff
@@ -660,6 +660,10 @@
                                                 forest=1
                                             [/movement_costs]
                                         [/effect]
+                                        [effect]
```

Comment:
Looks okay but the bonus description should be edited to reflect the decrement in max XP required.

Reply:
The current description is not too accurate:

> This massive book is filled with the Lore and Legends of the Wose. Any who read it learn deeply of the ways of the twig and leaf, the tree and the forest.

> "Upon finishing the book, $unit.name| has a much greater understanding of the forest, its nature and ways, and all the creatures within it. Unfortunately, as the last page was turned, the book crumbled to dust.

Keywords: understanding -> intelligence; forest -> ambush. But they are both long shots.